tests/commands/test_invoke_labelanalysis.py

```diff
@@ -332,20 +343,38 @@ def test_invoke_label_analysis_dry_run_with_output_path(
                     ],
                     obj={},
                 )
-                labels_file = Path("ats_output_path")
+                print(result)
+                print(result.output)
+                ats_output_path = "ats_output_path"
+                labels_file = Path(ats_output_path)
+                skip_labels_file = Path(ats_output_path + "_skipped")
+                json_output = Path(ats_output_path + ".json")
                 assert labels_file.exists() and labels_file.is_file()
+                assert skip_labels_file.exists() and skip_labels_file.is_file()
+                assert json_output.exists() and json_output.is_file()
                 with open(labels_file, "r") as fd:
                     assert fd.readlines() == [
-                        "--labels test_absent test_global test_in_diff\n"
+                        '--labels "test_absent" "test_global" "test_in_diff"\n'
+                    ]
+                with open(skip_labels_file, "r") as fd:
+                    assert fd.readlines() == ['--labels "test_present"\n']
+                with open(json_output, "r") as fd:
+                    assert fd.readlines() == [
+                        '{"runner_options": ["--labels"], "ats_tests_to_run": ["\\"test_absent\\"", "\\"test_global\\"", "\\"test_in_diff\\""], "ats_tests_to_skip": ["\\"test_present\\""]}\n'
```

Comment:
We recently had a customer run into trouble because one of their parametrized tests looked something like `path/to/test::TestClass::test_name[option -or another]`. When those were piped into `pytest` it assumed the `-or` was a command line option for pytest itself and failed.

That's why we are now double quoting all labels.